infer.py

```diff
@@ -59,7 +59,8 @@ def parsed_args():
 def eval_folder(path_to_images, output_dir, paint=False):
     images = [path_to_images / Path(i) for i in os.listdir(path_to_images)]
     for img_path in images:
-        img = pil_image_loader(img_path, task="x", domain="val")
+        # img = pil_image_loader(img_path, task="x", domain="val")
+        img = pil_image_loader(img_path, task="x")
```

Comment:
minor thing here but do we actually want to keep th pil image loader just for eval.py ? Or should we change everything to tensor loader?

Reply:
ahhh yeah we should probably use tensor_loader, would be a cleaner code

Reply:
Agreed